assets/javascripts/lib/variables-composer.js

```diff
@@ -0,0 +1,19 @@
+export default function extractVariablesFromComposerModel(model) {
+  if (model) {
+    return {
+      my_username: model.get("user.username"),
+      my_name: model.get("user.name"),
+      topic_title: model.get("topic.title"),
+      topic_url: model.get("topic.url"),
+      original_poster_username: model.get("topic.details.created_by.username"),
+      original_poster_name: model.get("topic.details.created_by.name"),
+      reply_to_username: model.get("post.username"),
+      reply_to_name: model.get("post.name"),
+      last_poster_username: model.get("topic.last_poster_username"),
+      reply_to_or_last_poster_username:
+        model.get("post.username") || model.get("topic.last_poster_username"),
+    };
+  }
+
+  return {};
```

Comment:
This is minor but whenever possible (i.e. in most cases) we can now avoid `get` and rely on native getters (and `?.` if needed)

So this could look probably like that

```suggestion
  if (!model) {
    return {};
  }

  return {
    my_username: model.user.username,
    my_name: model.user.name,
    topic_title: model.topic?.title,
    topic_url: model.topic?.url,
    original_poster_username: model.topic?.details.created_by.username,
    original_poster_name: model.topic?.details.created_by.name,
    reply_to_username: model.post?.username,
    reply_to_name: model.post?.name,
    last_poster_username: model.topic?.last_poster_username,
    reply_to_or_last_poster_username:
      model.post?.username || model.topic?.last_poster_username,
  };
```